eth/config.go

```diff
@@ -95,11 +95,12 @@ type Config struct {
 	TrieTimeout        time.Duration
 
 	// Mining-related options
-	Etherbase    common.Address `toml:",omitempty"`
-	MinerThreads int            `toml:",omitempty"`
-	MinerNotify  []string       `toml:",omitempty"`
-	ExtraData    []byte         `toml:",omitempty"`
-	GasPrice     *big.Int
+	Etherbase        common.Address `toml:",omitempty"`
+	MinerThreads     int            `toml:",omitempty"`
+	MinerNotify      []string       `toml:",omitempty"`
+	ExtraData        []byte         `toml:",omitempty"`
+	GasPrice         *big.Int
+	RecommitInterval time.Duration
```

Comment:
Please call this `MinerRecommit`. Bonus points if you rename `ExtraData` and `GasPRice` too to have the `Miner` prefix. Long term we could separate these out into an own config struct.